steer_max: 0.7853981633974483
rover_dimensions:
  d1: 0.184
  d2: 0.267
  d3: 0.267
  d4: 0.2556
  wheel_radius: 0.075
motor_drive_noload_rpm: 223
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
front_left_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: front_left_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
front_right_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: front_right_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
middle_left_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: middle_left_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
middle_right_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: middle_right_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
back_left_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: back_left_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
back_right_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: back_right_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
front_left_corner_controller:
  type: effort_controllers/JointPositionController
  joint: front_left_corner_joint
  pid:
    p: 50
    i: 20
    d: 1
front_right_corner_controller:
  type: effort_controllers/JointPositionController
  joint: front_right_corner_joint
  pid:
    p: 50
    i: 20
    d: 1
back_left_corner_controller:
  type: effort_controllers/JointPositionController
  joint: back_left_corner_joint
  pid:
    p: 50
    i: 20
    d: 1
back_right_corner_controller:
  type: effort_controllers/JointPositionController
  joint: back_right_corner_joint
  pid:
    p: 50
    i: 20
    d: 1
